packages/discovery-provider/src/api/v1/playlists.py

```diff
@@ -186,6 +186,8 @@ def get(self, playlist_id):
             playlist_id=playlist_id,
         )
         if playlist:
+            tracks = get_tracks_for_playlist(playlist_id, current_user_id)
+            playlist["tracks"] = tracks
```

Comment:
is it not `playlist_contents["tracks"]`?

Reply:
there's this mapping here https://github.com/AudiusProject/audius-protocol/blob/main/packages/discovery-provider/src/api/v1/helpers.py#L520-L524